GISMentors courses: fix links and update description (#45)
OSGeo/grass-tutorials: content/tutorials/external/GISMentors_Jena.qmd

  eval: false
---

> # GRASS GIS Workshop in Jena

Reviewer: ```suggestion
# GRASS Workshop in Jena
```

> A course taught at the University of Jena: Automated Earth Observation Processing Work Flows with GRASS GIS and WPS

Reviewer: ```suggestion
A course taught at the University of Jena: Automated Earth Observation Processing Work Flows with GRASS and WPS
```

> The idea of this course is to teach about spatio-temporal data processing with focus on satellite data including automated processing workflows provided as web services. Sample dataset used in this course includes LiDAR Data and Earth Observation Time Series. Automated workflows with open source GRASS GIS software and Python programming language.

Reviewer: ```suggestion
The idea of this course is to teach about spatio-temporal data processing with a focus on satellite data, including automated processing workflows provided as web services. The sample dataset used includes LiDAR data and Earth Observation time series. Workflows are automated with the open source GRASS GIS software and Python programming language.
```
